getting-started.md

```diff
@@ -0,0 +1,96 @@
+# Getting started with Kontena Pharos
+
+Pharos allows you to bootstrap and manage your Kubernetes clusters super easily. You can get started in few minutes by following these steps.
+
+1. Setup Pharos CLI tooling
+2. Prepare the Nodes for the Kubernetes Cluster
+3. Creating the Cluster Configuration File
+4. Bootstrapping Kubernetes cluster using Pharos
+5. Interacting with your Kubernetes Cluster
+
+
+## Setup Pharos CLI tooling
+
+Follow these easy steps to setup Pharos CLI tooling. For more detailed instructions and explanations, see [CLI installation docs](install.md).
+
+First we need to install our Pharos version switcher tool:
+```
+$ curl -s https://get.pharos.sh | bash
+```
+```
+$ chpharos login
+Log in using your Kontena Account credentials
+Visit https://account.kontena.io/ to register a new account.
+Username: adam
+Password:
+Logged in.
+```
+
+With `chpharos` we can easily manage the actual Pharos CLI tool:
+```
+$ chpharos install latest --use
+```
+
+Now you should have the tooling ready, we can continue to next phase.
+
+# Prepare nodes for Kubernetes cluster
+
+Most of the Kubernetes cluster components are launched using containers (Docker or CRI-O) on a Linux distro. You can use most common Linux distros, we [support many of them](requirements.md).
+
+> **Tip:** In our GitHub repo, there's ready made examples for setting up the cluster using Vagrant boxes. This is an easy way to get started if you do not have any other infrastructure available to play with. See https://github.com/kontena/pharos-cluster/tree/master/examples/vagrant/centos7
+
+# Create the cluster configuration file
+
+With Pharos, the cluster is [configured](configuration.md) using a yaml file. In this example we setup simple 1+1 cluster, one node acts as the control plane and one as pure worker node.
+
+So open up your favourite editor, and type something similar as in the example below. Once done, save the file as `cluster.yml`. Naturally you need to adjust the example to match your infrastructure at hand.
+
+```yaml
+hosts:
+  - address: 192.168.110.100
+    user: vagrant
+    role: master
+    ssh_key_path: ~/.ssh/my_key
+  - address: 192.168.110.101
+    user: vagrant
+    role: worker
+    ssh_key_path: ~/.ssh/my_key
+network: {} # Use Weave networking with default config
+addons:
+  ingress-nginx:
+    enabled: true # Enable Nginx ingress controller
```

Comment:
Should we include Lens here?

Reply:
My thinking was to keep the first getting started really simple. Lens comes with a lot of options that might confues new comers.